top/include/lk/utils.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (C) 2020-2023 Google LLC.
+ */
+
+#pragma once
+
+#define ARRAY_SIZE(_a) (sizeof(_a) / sizeof(*(_a)))
+
+#define KB (1024UL)
```

Comment:
There are some uses of this elsewhere, so will probably need to smush the other defines into this one. Will do.